phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/controllers/payments/Scan.kt

```diff
@@ -14,6 +14,7 @@ object Scan {
 
     sealed class BadRequestReason {
         data class ChainMismatch(val myChain: Chain, val requestChain: Chain?): BadRequestReason()
+        object InvalidLnUrl: BadRequestReason()
```

Comment:
Yes! Good idea. This was bothering me too.

> I think that Lnurl maybe the best option because it's simple and it works well for variable names

Agreed.

And it would be good to have consistency within our code. Especially since there isn't consistency elsewhere:

- [io.ktor.http.Url](https://api.ktor.io/ktor-http/ktor-http/io.ktor.http/-url/index.html)
- [io.ktor.http.URLBuilder](https://api.ktor.io/ktor-http/ktor-http/io.ktor.http/-u-r-l-builder/index.html)
- [Swift.URL](https://developer.apple.com/documentation/foundation/url)

:joy: